beautifultable/utils.py

```diff
@@ -55,7 +61,14 @@ def get_output_str(item, detect_numerics, precision, sign_value):
 
 def ansilen(item):
     """Returns the length of the string after stripping ansi escape sequences"""
-    return len(ANSI_REGEX.sub('', item))
+    total_width = 0
+    for char in item:
+        char_type = east_asian_width(char)
```

Comment:
Getting this error on Python version 2.7
https://travis-ci.org/pri22296/beautifultable/jobs/462833270
`east_asian_width() argument 1 must be unicode, not str`